pkg/server/convert.go

```diff
@@ -1,4 +1,4 @@
-package translate
+package server
```

Comment:
package server? didn't you mean service?

Reply:
this one debatable, maybe you right, @janishorsts what you think, service or server ?

Reply:
keep `server`